docs/modelserving/v1beta1/serving_runtime.md

```diff
@@ -21,18 +21,31 @@ After models are deployed with InferenceService, you get all the following serve
 - Out-of-the-box metrics
 - Ingress/Egress control
 
-| Model Serving Runtime  | Exported model| Prediction Protocol | HTTP | gRPC | Versions | Examples                             |
+
+---
+
+The table below identifies each of the model serving runtimes supported by KServe. The HTTP and gRPC columns indicate the prediction protocol version that the serving runtime supports. The KServe prediction protocol is noted as either "v1" or "v2". Some serving runtimes also support their own prediction protocol, these are noted with an `*`. The default serving runtime version column defines the source and version of the serving runtime - MLServer, KServe or its own. These versions can also be found in the [runtime kustomization YAML](https://github.com/alexagriffith/kserve/blob/master/config/runtimes/kustomization.yaml). All KServe native model serving runtimes use the current KServe release version (v0.10). The supported framework version column lists the **major** version of the model that is supported. These can also be found in the respective [runtime YAML](https://github.com/alexagriffith/kserve/tree/master/config/runtimes) under the `supportedModelFormats` field. For model frameworks using the KServe serving runtime, the specific default version can be found in [kserve/python](https://github.com/alexagriffith/kserve/tree/master/python). In a given serving runtime directory the setup.py file contains the exact model framework version used. For example, in [kserve/python/lgbserver](https://github.com/alexagriffith/kserve/tree/master/python/lgbserver) the [setup.py](https://github.com/alexagriffith/kserve/blob/master/python/lgbserver/setup.py) file sets the model framework version to 3.3.2, `lightgbm == 3.3.2`.
```

Comment:
Please fix the reference urls which point to you personal GitHub repo :)

Reply:
ah woops good catch - must have been looking at my fork

Reply:
done